setup.py

```diff
@@ -78,6 +79,8 @@ def get_version(verbose=1, filename='src/qtt/version.py'):
       license='MIT',
       package_dir={'': 'src'},
       packages=find_packages(where='./src'),
+#      package_data={'': ['exampledata/*.*', 'exampledata/*/*.*', 'exampledata/*/*/*.*', 'exampledata/*/*/*/*.*']},
+      include_package_data = True,
       data_files=[('bin',
```

Comment:
If and only if we need to install some example data, we can also add it to data files, instead of adding manifest

Reply:
Only adding to the data files doesn't work, it seems we need the manifest as well. Documentation on the exact behavior is hard to find though.